src/sc/sidechain.cpp

```diff
@@ -142,11 +142,12 @@ bool CSidechain::CheckCertTiming(int certEpoch, int referencedHeight, const CCoi
                     __func__, __LINE__, certEpoch, lastTopQualityCertReferencedEpoch + 1);
         }
 
-        // Check ordering of references
-        if (referencedHeight <= lastReferencedHeight)
+        // Check that every certificate references a block whose commitment tree includes the previous certificate.
+        // This also implies referencedHeight > lastReferencedHeight.
+        if (referencedHeight < lastInclusionHeight)
         {
-            return error("%s():%d - ERROR: certificate cannot be accepted, cert height (%d) not greater than last (%d)\n",
-                __func__, __LINE__, referencedHeight, lastReferencedHeight);
+            return error("%s():%d - ERROR: certificate cannot be accepted, cert height (%d) not greater than last certificate inclusion height (%d)\n",
```

Comment:
I think `not greater than` should be restated as `lesser than`

Reply:
Yes, makes sense, thanks.